PoGo.Necrobot.Window/Model/GymViewModel.cs

```diff
@@ -30,19 +30,41 @@ public string GymIcon
             {
                 string fortIcon = "";
                 bool isRaid = false;
+                bool asBoss = false;
+
                 try
                 {
-                    if (!string.IsNullOrEmpty(fort.RaidInfo.RaidPokemon.PokemonId.ToString()))
+                    if (fort.RaidInfo.RaidBattleMs > 0)
                         isRaid = true;
                 }
                 catch
                 {
                     isRaid = false;
                 }
-                string gymStat = isRaid ? "-raid" : "";
+
+                try
+                {
+                    if (!string.IsNullOrEmpty(fort.RaidInfo.RaidPokemon.PokemonId.ToString()))
+                        asBoss = true;
+                }
+                catch
+                {
+                    asBoss = false;
+                }
+
+                string gymStat = isRaid ? "-raid" : null;
+                string gymBoss = asBoss ? $"https://cdn.rawgit.com/Necrobot-Private/PokemonGO-Assets/master/pokemon/{(int)fort.RaidInfo.RaidPokemon.PokemonId}.png" : null;
+
+                // Solution to overlay 2 images in string mode ????
+                // forticon + gymBoss or create asset 251 gyms red + 251 gyms blue + 251 neutral +251 yellow !!!!
+                // WFP is compatible to var image ??? 
+
```

Comment:
Hence it is complicated to recover each slashed pokemon or gym with boss which gives a lot of images in typo assets

Pidgey-slashed.png .... etc
Gym-instinc-lapras.png (exemple boos) etc.

This makes a lot of image if not combine images to have idea in total the game has 700 and such pokemons what of the neutral gym x 251 present x 4 instint mystic value this knowing that we are still only in the 2 generation of pokemons